lib/routes/password.js

```diff
@@ -408,28 +409,26 @@ module.exports = function (
           )
           .then(
             function (passwordForgotToken) {
-              return getGeoData(ip)
-                .then(
-                  function (geoData) {
-                    return mailer.sendRecoveryCode(
-                      [],
-                      passwordForgotToken,
-                      userAgent.call({
-                        code: passwordForgotToken.passCode,
-                        token: passwordForgotToken,
-                        service: service,
-                        redirectTo: request.payload.redirectTo,
-                        resume: request.payload.resume,
-                        acceptLanguage: request.app.acceptLanguage,
-                        flowId: flowId,
-                        flowBeginTime: flowBeginTime,
-                        ip: ip,
-                        location: geoData.location,
-                        timeZone: geoData.timeZone
-                      }, request.headers['user-agent'], log)
-                    )
-                  }
-                )
+              return P.all([getGeoData(ip), db.accountEmails(passwordForgotToken.uid)])
```

Comment:
should the `db.accountEmails(passwordForgotToken.uid)` be behind the secondary email feature flag?